grease/src/Grease/Macaw/PLT.hs

```diff
@@ -112,15 +112,18 @@ pltStubSymbols pltStubInfo loadOptions ehi =
 
   withinPltSection ::
     (Ord (Elf.ElfWordType w), Num (Elf.ElfWordType w)) =>
-    MM.MemWord w -> Bool
+    MM.MemWord w ->
+    Bool
   withinPltSection addr =
     case Elf.findSectionByName ".plt" elf of
       [section] -> withinSection addr section
       _ -> False
 
   withinSection ::
     (Ord (Elf.ElfWordType w), Num (Elf.ElfWordType w)) =>
-    MM.MemWord w -> Elf.ElfSection (Elf.ElfWordType w) -> Bool
+    MM.MemWord w ->
+    Elf.ElfSection (Elf.ElfWordType w) ->
+    Bool
   withinSection addr section =
     let addr' :: Elf.ElfWordType w
         addr' = fromIntegral addr
```

Comment:
Why did formatting change in this file? Is there something wrong with our Formolu setup?

Reply:
both are valid under formulo maybe i hit enter by accident im not sure?

Reply:
idk im unsure actually

Reply:
Okay, let's just keep an eye on it.